### PR TITLE
Added universal expressions (short form functions): $ex

### PR DESCRIPTION
**Pull request by dbuezas**
`$fn ({getFromConfig, xs, ...etc...}) => getFromConfig("title")`
equivalent to 
`$f get("title")`

(all other params also available)

### CODE REVIEW

**Comment by dbuezas** [comment]
What do you think about this?
* `$fn` is very explicit and easy to understand, but I find it too verbose. Having to state the parameters and whatnot is redundant.
* I'm considering this short form `$f`, that automatically adds this prelude:
`({ getFromConfig, get, hass, vars, path, css_vars, xs, ys, statistics, states, meta })=>` 
* and also `get` is an alias of getFromConfig

I think that it's good to keep $fn not only to avoid further breaking changes, but also because it explains $f in a sense.

@FrnchFrgg @zanna-37 you've both been very critical and constructive, I'd really appreciate your opinions&thoughts on this

**Comment by dbuezas** [comment]
Here's the skynet's opinion:
image
image

`$expr` and `$ex` are really not bad. (I'll hold my urge to pick`$jsex`)

**Comment by dbuezas** [comment]
I used this a bit today and it is quite convenient, but I'll rename it to `$ex`, a "universal expression".

**Comment by FrnchFrgg** [comment]
Indeed, something like `$ex` or `$expr` is what I would recommend, since it corresponds to the expression part of a lambda function. I slightly prefer the longer form, but not much.

File: src/parse-config/parse-config.ts [hunk 1]
```diff
@@ -112,8 +114,15 @@ class ConfigParser {
       await this.fetchDataForEntity(entityPath);
     }
 
-    if (typeof value === "string" && value.startsWith("$fn")) {
-      value = myEval(value.slice(3));
+    if (typeof value === "string") {
+      if (typeof value === "string" && value.startsWith("$ex")) {
```
Comment (FrnchFrgg) [inline src/parse-config/parse-config.ts:118]: Why do you check again for the type of `value` ?

Comment (dbuezas) [reply src/parse-config/parse-config.ts:118]: To be extra sure?
Thanks, I fixed that now.

**Comment by dbuezas** [comment]
> I slightly prefer the longer form, but not much.

I went with $ex partly to keep it short, but mostly because $fn also had 2 letters. I've been using it and it is quite convenient :)

**Pull request merged**